packages/blitz-dom/src/document.rs

```diff
@@ -1132,8 +1134,8 @@ impl Document {
     pub fn scroll_viewport_by(&mut self, x: f64, y: f64) {
         let content_size = self.root_element().final_layout.size;
         let new_scroll = (self.viewport_scroll.x - x, self.viewport_scroll.y - y);
-        let window_width = self.viewport.window_size.0 as f64 / self.viewport.scale() as f64;
-        let window_height = self.viewport.window_size.1 as f64 / self.viewport.scale() as f64;
+        let window_width = self.viewport.window_size.0 as f64 / self.viewport.hidpi_scale as f64;
+        let window_height = self.viewport.window_size.1 as f64 / self.viewport.hidpi_scale as f64;
```

Comment:
Was this change intentional? It's not equivalent to the previous code (which included the `zoom`).

If not, then IMO this is actually quite a good argument for having the `hidpi_scale` and` zoom` field private as they were before, as it prevents this kind of accidental usage of the wrong value.

If it is, then can you provide a justification for the change?